src/main/java/it/pagopa/transactions/utils/LogoMappingUtils.java

```diff
@@ -37,11 +34,15 @@ public URI getLogo(AuthorizationRequestData authRequestedData) {
                 URI unknown = npgPaymentCircuitLogoMap.get(BrandLogoConfig.UNKNOWN_LOGO_KEY);
                 yield npgPaymentCircuitLogoMap.getOrDefault(authRequestedData.brand(), unknown);
             }
-            case WalletAuthRequestDetailsDto ignored1 -> {
+            case WalletAuthRequestDetailsDto ignored -> {
                 URI unknown = npgPaymentCircuitLogoMap.get(BrandLogoConfig.UNKNOWN_LOGO_KEY);
                 yield npgPaymentCircuitLogoMap.getOrDefault(authRequestedData.brand(), unknown);
             }
-            default -> throw new InvalidRequestException("Authorization request detail type not valid");
+            case ApmAuthRequestDetailsDto ignored -> {
+                URI unknown = npgPaymentCircuitLogoMap.get(BrandLogoConfig.UNKNOWN_LOGO_KEY);
```

Comment:
Just for info, do we have a dedicated task to retrive this kind of payment's logo info?

Reply:
Actually this should fish the url from the map of those apm registered